prow/cmd/deck/template/index.html

```diff
@@ -2,7 +2,7 @@
 
 {{define "scripts"}}
 <script type="text/javascript" src="/static/prow_bundle.min.js"></script>
-<script type="text/javascript" src="prowjobs.js?var=allBuilds&omit=annotations,labels,decoration_config,pod_spec"></script>
+<script type="text/javascript" src="prowjobs.js?var=allBuilds&omit=annotations,labels,decoration_config"></script>
```

Comment:
Including the pod_spec in the results seems to increase the GET time by >50% on average according to some checks I did with `time curl`. That seems a little excessive given that we only need to know if there are multiple containers. Can we determine this more efficiently or perhaps only check if the logs button is actually clicked?

Reply:
I think I found a way to solve this issue by changing the way handleProwJobs() works in `deck/main.go.`  If instead of completely setting the podspec to nil when omitted we just set the podspec to be nothing but a list of empty containers, this gives us the information we need in order to achieve the same behavior.  From the testing that I've done locally with `time curl` it seems like the GET time is not nearly as horrendous as retrieving the entire podspec.